src/orgwarden/audit.py

```diff
@@ -2,12 +2,21 @@
 from orgwarden.repo_crawler import Repository
 
 
-def audit_repository(repo: Repository) -> int:
+def audit_repository(
+    repo: Repository,
+    gh_pat: str | None,
+) -> int:
     """
-    Runs RepoAuditor against the specified repository and returns a tuple containing the exit code and the stdout if `capture` is set to True.
+    Runs RepoAuditor against the specified repository and returns the resulting exit code.
     """
+
+    BASE_COMMAND = f"uv run repo_auditor --include GitHub --GitHub-url {repo.url}"
```

Comment:
This code feels redundant as written, as `command` is set multiple times when `gt_pat` is not None.  
  
Consider:

```
command = f"uv run repo_auditor --include GitHub --GitHub-url {repo.url}"
if gh_pat is not None:
    command += f" --GitHub-pat {gh_pat}"
```